README.md

```diff
@@ -373,6 +373,22 @@ debug-logs = true
 
 Run `./cloud-sql-proxy --help` for more details.
 
+### Config environment variables
+
+The proxy supports configuration through environment variables. 
+Each environment variable uses "CSQL_PROXY" as a prefix and is 
+the uppercase version of the flag using underscores as word delimiters. For 
+example, the `--auto-iam-authn` flag may be set with the environment variable 
+`CSQL_PROXY_AUTO_IAM_AUTHN`. An invocation of the Proxy using environment 
+variables would look like the following: 
+
+```shell
+CSQL_PROXY_AUTO_IAM_AUTHN=true \ 
+    ./cloud-sql-proxy my-project:us-central1:my-db-server 
```

Comment:
let's use either `INSTANCE_CONNECTION_NAME` or `myproject:my-region:myinstance` for consistency
```suggestion
    ./cloud-sql-proxy <INSTANCE_CONNECTION_NAME>
```